xml/deploy_additional.xml

```diff
@@ -300,6 +300,55 @@
       </listitem>
      </itemizedlist>
     </note>
+    <important>
+     <title>Incorrectly reported image sizes</title>
+     <para>
+      A bug in older kernels caused that an RBD target reported an image size
+      one block (512B) larger than it actually was. Therefore, the LUN could
+      write one additional block of the image. Although this bug is fixed now,
```

Comment:
nit: `Although this bug is now fixed, ...`

Also, I think `512B` should be spelled out as `512 bytes`.